libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/feedback/FeedbackAdapter.java

```diff
@@ -24,18 +22,14 @@
   private List<FeedbackItem> feedbackItems = new ArrayList<>();
 
   FeedbackAdapter(Context context) {
-    feedbackItems.add(new FeedbackItem(context.getString(R.string.feedback_not_allowed),
-      R.drawable.ic_no_turn_allowed, FEEDBACK_TYPE_NOT_ALLOWED, EMPTY_FEEDBACK_DESCRIPTION));
     feedbackItems.add(new FeedbackItem(context.getString(R.string.feedback_road_closure),
       R.drawable.ic_road_closed, FEEDBACK_TYPE_ROAD_CLOSED, EMPTY_FEEDBACK_DESCRIPTION));
-    feedbackItems.add(new FeedbackItem(context.getString(R.string.feedback_report_traffic),
-      R.drawable.ic_traffic, FEEDBACK_TYPE_REPORT_TRAFFIC, EMPTY_FEEDBACK_DESCRIPTION));
+    feedbackItems.add(new FeedbackItem(context.getString(R.string.feedback_not_allowed),
+      R.drawable.ic_not_allowed, FEEDBACK_TYPE_NOT_ALLOWED, EMPTY_FEEDBACK_DESCRIPTION));
     feedbackItems.add(new FeedbackItem(context.getString(R.string.feedback_confusing_instruction),
       R.drawable.ic_confusing_directions, FEEDBACK_TYPE_CONFUSING_INSTRUCTION, EMPTY_FEEDBACK_DESCRIPTION));
-    feedbackItems.add(new FeedbackItem(context.getString(R.string.feedback_general_issue),
-      R.drawable.ic_map_error, FeedbackEvent.FEEDBACK_TYPE_OTHER_MAP_ISSUE, EMPTY_FEEDBACK_DESCRIPTION));
```

Comment:
We should remove `FEEDBACK_TYPE_OTHER_MAP_ISSUE` constant from `FeedbackEvent` if not used anymore.